lib/indexer.rb

```diff
@@ -0,0 +1,121 @@
+module Argo
+  class Indexer
+    SimpleLogJob = Struct.new(:log_val) do
+      def perform
+        sleep(rand 5)
+        job_logger = Logger.new(File.open("tmp/simple_job.log"))
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.